### PR TITLE
Adding updated Hotel Service 2 documentation

### PR DESCRIPTION
**Pull request by wojtekkedzior**
- adding a new side panel entry for Hotel Service 2 documentation
- adding new, incomplete, content.  This will be followed up with more pull requests
- the old page will remain in place as we have already shared the URL, but I will update it and list a link to this new section

### CODE REVIEW

File: api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
[the remark this answers is not in the document]
Comment (wojtekkedzior) [reply api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown]: Good point. Fixed with latest commit.

File: _data/sidebars/api-reference.yml [hunk 2]
```diff
@@ -88,6 +88,31 @@
         url: /api-reference/direct-connects/hotel/post-reservation-update.html
       - title: Hotel Direct Connect 2.0
         url: /api-reference/direct-connects/hotel-service.html
+- title: Direct Connect: Hotel Service 2
```
[the remark this answers is not in the document]
Comment (wojtekkedzior) [reply _data/sidebars/api-reference.yml]: Good catch. Fixed with latest commit.

**Comment by bhague1281** [comment]
Looks good to me. A couple things you may want to address:

1. Appendix page doesn't exist. I removed it from the sidebar so our link-checker won't break, but feel free to add the page in a new PR.
2. Any details on what will happen to the old Hotel Direct Connects 2.0 page? (This one: https://developer.concur.com/api-reference/direct-connects/hotel-service.html)

**Comment by wojtekkedzior** [comment]
Good points.  I will create an Appendix page in the future.  With regards to the old page - since it has been shared with some vendors I will update it by removing all the content and providing links to the new page.  Is that reasonable?

**Comment by bhague1281** [comment]
Sounds good 👍

**Comment by wojtekkedzior** [comment]
When can we expect for this pull request to be merged?  I'm still adding content, but I would like to get this one big chunk merged before adding more smaller ones.

**Comment by bhague1281** [comment]
I'll merge now. @elhansen748 or I merge to the live site weekly.

**Pull request merged**